### PR TITLE
Black/flake8 fixups

### PR DESCRIPTION
**Pull request by Nubly**
Simple [`black`](https://black.readthedocs.io/en/stable/)/[`flake8`](https://flake8.pycqa.org/en/latest/) fixups, should be just formatting and some simple stylistic improvements

### CODE REVIEW

File: client.py [hunk 1]
```diff
@@ -5,57 +5,60 @@
 from discord.ext import commands
 
 import data
-from cogs import *
```
Comment (Nubly) [inline client.py:8]: Seemingly unused?

File: client.py [hunk 2]
```diff
@@ -5,57 +5,60 @@
...
-            intents=discord.Intents.default())
+        super().__init__(command_prefix="$", intents=discord.Intents.default())
         self.data = data.Data()
         self.set_env_vars()
 
     def set_env_vars(self):
         self.TOKEN, self.GUILD_NAME = self.data.get_env_vars()
-        if self.TOKEN == None or self.GUILD_NAME == None:
-            print("ERROR in .env file, double check format in top-level README.md")
+        if not all(self.TOKEN or self.GUILD_NAME):
```
Comment (Nubly) [inline client.py]: Oops, this needs to be something along the lines of...
```suggestion
        if not all([self.TOKEN, self.GUILD_NAME]):
```
actually

File: client.py [hunk 3]
```diff
@@ -5,57 +5,60 @@
...
+            )
             exit()
 
     async def setup_hook(self):
-        #Load cogs
-        for filename in os.listdir('./cogs'):
-            if filename.endswith('.py') and filename != '__init__.py':
+        # Load cogs
+        for filename in os.listdir("./cogs"):
+            if filename.endswith(".py") and filename != "__init__.py":
```
Comment (Nubly) [inline client.py:28]: Consider using [`pathlib.Path.glob()`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob) here perhaps

File: cogs/Speech.py [hunk 4]
```diff
@@ -11,48 +11,61 @@
...
+    ):
+        # Extract the actual voice object (this lets us modify the voice)
         voice_list = voices()
         voice_obj = voice_list[0]
         for voice in voices():
             if voice.name == my_voice:
                 voice_obj = voice
 
-        #Make it wacky if so desired
+        # Make it wacky if so desired
```
Comment (Nubly) [inline cogs/Speech.py:68]: 🤪 quirked up

File: cogs/Util.py [hunk 5]
```diff
@@ -2,20 +2,25 @@
...
 import data
 
+
 class Util(commands.Cog):
     def __init__(self, bot: commands.Bot):
         self.bot = bot
         self.data = data.Data()
 
-    #Dms are assumed to be elevenlabs keys
+    # Dms are assumed to be elevenlabs keys
```
Comment (Nubly) [inline cogs/Util.py]:
```suggestion
    # DMs are assumed to be ElevenLabs keys
```

File: cogs/Util.py [hunk 6]
```diff
@@ -2,20 +2,25 @@
...
         if message.author == self.bot.user:
             return
         if not message.guild:
             self.data.save_eleven_labs_key(message.author.id, message.content)
-            await message.channel.send("Hello " + message.author.name + ", your elevenlabs id has been added")
+            await message.channel.send(
+                "Hello "
+                + message.author.name
+                + ", your elevenlabs id has been added"
+            )
```
Comment (Nubly) [inline cogs/Util.py]:
```suggestion
            await message.channel.send(
                f"Hello {message.author.name}, your ElevenLabs ID has been added."
            )
```
unsure if f-string will work here but I believe so

File: data.py [hunk 7]
```diff
@@ -2,30 +2,34 @@
...
+        with open(self.key_filename, "wb") as file:
             pickle.dump(eleven_labs_key_dict, file)
 
     """
-    If user is registered, returns the tuple (True, <key associated with the user>)
-    If user is  not registered, returns the tuple (False, None)
+    If user is registered, returns the tuple:
+        (True, <key associated with the user>)
+    If user is not registered, returns the tuple:
+        (False, None)
```
Comment (Nubly) [inline data.py:30]: [`Typing`](https://docs.python.org/3/library/typing.html) might be nice for this, as well as for all the other files and whatnot

File: client.py (hunk 3 above)
Comment (rnofal1) [reply client.py:28]: You got it boss man.

File: data.py (hunk 7 above)
Comment (rnofal1) [reply data.py:30]: Agreed

File: client.py (hunk 1 above)
Comment (rnofal1) [reply client.py:8]: Yup.

**Pull request merged**